File: uri/c/1008.c
#include <stdio.h>

int main() {

    int numeroEmpregado, horasTrabalhadas;
    float salarioPorHora, salario;

    scanf("%d", &numeroEmpregado);
    scanf("%d", &horasTrabalhadas);
    scanf("%f", &salarioPorHora);

    salario = horasTrabalhadas * salarioPorHora;

    printf("NUMBER = %d\n", numeroEmpregado); 
    printf("SALARY = U$ %.2f\n", salario); 

    return 0;

}

File: uri/go/1008.go
package main

import (
	"fmt"
)

func main() {

	var employeeNumber int
	var workedHours int
	var salaryPerHour float64

	fmt.Scanf("%d\n%d\n%f", &employeeNumber, &workedHours, &salaryPerHour)
	fmt.Printf("NUMBER = %d\n", employeeNumber)

	salary := float64(workedHours) * salaryPerHour
	fmt.Printf("SALARY = U$ %.2f\n", salary)
	
}